client/src/annoMatrix/annoMatrix.ts

```diff
@@ -475,26 +470,23 @@ Return cache keys for columns associated with this query.  May return
   /**
    ** Private interfaces below.
    **/
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'field' implicitly has an 'any' type.
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  _resolveCachedQueries(field, queries) {
-    return (
-      queries
-        // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'query' implicitly has an 'any' type.
-        .map((query) =>
-          _whereCacheGet(this._whereCache, this.schema, field, query).filter(
-            // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'cacheKey' implicitly has an 'any' type.
-            (cacheKey) =>
-              cacheKey !== undefined && this._cache[field].hasCol(cacheKey)
-          )
+  _resolveCachedQueries(
+    field: Field,
+    queries: Query[]
+  ): string[] | number[] | [undefined] {
```

Comment:
You can remove `[undefined]` from this return type, as they are filtered out on line 483.   That will simply typing upstream.

Reply:
Updated [annoMatrix.ts:: _resolveCachedQueries](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L474)